charts/workflows/charts/mta/templates/00-mta-operator.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
```

Comment:
numbers are meaningless. helm chart reads and sorts all of the resources based on its order.
this can be named simply mta-operator.yaml.
same for the other resource. not going to insist on this.

Reply:
in know but I like to have them sorted in which order I would do if I were to do it manually
The 01 and 02 etc are generated so I do not know about them but with 00 I directly understand that they should be apply first